lib/lita/handlers/reviewme.rb

```diff
@@ -96,16 +96,16 @@ def comment_on_github(response)
         end
       end
 
-      def mention_reviewer(response)
+      def mention_reviewer(response, room: get_room(response))
         url = response.matches.flatten.first
-        reviewer = next_reviewer
+        reviewer = next_reviewer(room)
         response.reply(chat_mention(reviewer, url))
       end
 
       private
 
-      def next_reviewer
-        redis.rpoplpush(REDIS_LIST, REDIS_LIST)
+      def next_reviewer(room)
+        ns_redis(room).rpoplpush(REDIS_LIST, REDIS_LIST)
```

Comment:
Am I correct to assume, that `ns_redis(room)` would serialize the room to redis and then use it as a key? Could you instead use a shorter key like `"room:#{room.id}". (That assumes that `room` has a unique id)